src/test/java/com/github/onsdigital/babbage/template/handlebars/helpers/resolve/DataHelpersTest.java

```diff
@@ -73,7 +73,7 @@ public void testDataHelpers_resolveTaxonomy_Navigation() throws Exception {
         mockDH.isPublication = false;
         mockDH.isNavigation = true;
 
-        CharSequence resolve = mockDH.resolveTaxonomy.apply(uri, options);
+        Object resolve = mockDH.resolveTaxonomy.apply(uri, options);
```

Comment:
The resolveTaxonomy field is static so it should be accessed using the class name:
```suggestion
        Object resolve = DataHelpers.resolveTaxonomy.apply(uri, options);
```